README.md

```diff
@@ -3,113 +3,31 @@
+- Read about [running the UI from git checkout](https://github.com/canonical/lxd-ui/blob/main/HACKING.md), tests and advanced setup.
     
 # Examples
 
-| Image Selection | Instances | Instances - Detail |
-|--|--|--|
-| ![imageselect](https://user-images.githubusercontent.com/45884264/216782621-54fa4b6e-84b0-4854-bb38-187eea6a9071.png) | ![instances](https://user-images.githubusercontent.com/1155472/217785833-199575fb-149a-47e3-a597-98178348f06b.png) | ![instancedetail](https://user-images.githubusercontent.com/1155472/217785830-8fddc190-7464-4cd3-9f22-ee3787bc47f9.png) |
+| Create an instance                                                                                  | Instance list                                                                                                  | Instance terminal                                                                                          |
+|-----------------------------------------------------------------------------------------------------|----------------------------------------------------------------------------------------------------------------|------------------------------------------------------------------------------------------------------------|
+| ![0create](https://github.com/canonical/lxd-ui/assets/1155472/7f0c45a6-2ba2-4cc7-bd7c-c0ebca76d648) | ![1instance-overview](https://github.com/canonical/lxd-ui/assets/1155472/c71d2153-ea71-4ecb-ab25-fabcd6fb1e55) | ![2instance-term](https://github.com/canonical/lxd-ui/assets/1155472/c2b741e2-8806-4d4d-9a9a-f536f76a13b9) |
 
-| Snapshots | Terminal | Profiles - Detail |
-|--|--|--|
-| ![snapshots](https://user-images.githubusercontent.com/1155472/217785825-98036b93-9f0f-4dd6-af92-388ffa5d3a41.png) | ![terminal](https://user-images.githubusercontent.com/1155472/217785823-ec0ef782-b0d5-4d89-874e-9925ae6ba383.png) | ![profiledetail](https://user-images.githubusercontent.com/1155472/217785819-3929c3fa-14aa-4cac-a79a-1265d502555e.png) |
+| Graphic console                                                                                                | Storage                                                                                                | Cluster groups                                                                                                        |
+|----------------------------------------------------------------------------------------------------------------|--------------------------------------------------------------------------------------------------------|-----------------------------------------------------------------------------------------------------------------------|
+| ![3-instance-console](https://github.com/canonical/lxd-ui/assets/1155472/0f8d742d-3f9c-4906-90da-e740e8ff353b) | ![5storage](https://github.com/canonical/lxd-ui/assets/1155472/38d7b8ab-d652-4c18-b71e-0098efe73702)   | ![6cluster](https://github.com/canonical/lxd-ui/assets/1155472/85f61ef9-a45f-4b4a-abee-8fa9dfa69bd2) |
```

Comment:
Why do we have a screenshot for "Storage" - which is an "incomplete" page in terms of design - and none about "Profile list"?

Reply:
Changed it for a profile list:
![Screenshot from 2023-06-19 12-40-54](https://github.com/canonical/lxd-ui/assets/1155472/36a0f619-767f-4949-804d-061e5e28c87a)